stdlib/string.mli

```diff
@@ -32,21 +32,19 @@
   substring of [s] if [len >= 0] and [start] and [start+len] are
   valid positions in [s].
 
-  OCaml strings used to be modifiable in place, for instance via the
-  {!String.set} and {!String.blit} functions described below. This
-  usage is deprecated and only possible when the compiler is put in
-  "unsafe-string" mode by giving the [-unsafe-string] command-line
-  option (which is currently the default for reasons of backward
-  compatibility). This is done by making the types [string] and
-  [bytes] (see module {!Bytes}) interchangeable so that functions
-  expecting byte sequences can also accept strings as arguments and
-  modify them.
-
-  All new code should avoid this feature and be compiled with the
-  [-safe-string] command-line option to enforce the separation between
-  the types [string] and [bytes].
-
- *)
+  Note: OCaml strings used to be modifiable in place, for instance via
+  the {!String.set} and {!String.blit} functions described below. This
+  usage is only possible when the compiler is put in "unsafe-string"
+  mode by giving the [-unsafe-string] command-line option. This
+  compatibility mode makes the types [string] and [bytes] (see module
+  {!Bytes}) interchangeable so that functions expecting byte sequences
+  can also accept strings as arguments and modify them.
+
+  The difference between [bytes] and [string] was introduced in OCaml
```

Comment:
Looks good.  I wonder what word is better: "The difference between" or "The distinction between"...  Any native English speaker around?

Reply:
FWIW I'd use 'distinction' since there is for now still an equality underneath.

Reply:
I went for 'distinction', thanks!